keras_cv/layers/preprocessing/random_saturation_test.py

```diff
@@ -89,8 +90,4 @@ def test_with_unit8(self):
     def test_config(self):
         layer = preprocessing.RandomSaturation(factor=(0.3, 0.8))
         config = layer.get_config()
-        self.assertEqual(config["factor"], (0.3, 0.8))
-
-        layer = preprocessing.RandomSaturation(factor=0.5)
-        config = layer.get_config()
-        self.assertEqual(config["factor"], (0.0, 0.5))
+        self.assertTrue(isinstance(config["factor"], core.FactorSampler))
```

Comment:
Might want to do more strict validation here.

Reply:
Updated